pkg/vadmin/restore_points_query_vc.go

```diff
@@ -38,7 +38,9 @@ func (v *VClusterOps) ShowRestorePoints(ctx context.Context, opts ...showrestore
 		return restorePoints, err
 	}
 
-	s := showrestorepoints.Parms{}
+	s := showrestorepoints.Parms{
+		FilterOptions: &vops.ShowRestorePointFilterOptions{},
+	}
```

Comment:
Can we add this logic right in the `Make` function?

Reply:
yes